libdevcore/Common.h

```diff
@@ -294,6 +294,12 @@ enum class WithExisting: int
 	Kill
 };
 
+/// Get the current time in seconds since the epoch in UTC
+inline time_t getNowUTC()
```

Comment:
ultimately it must be converted to a `u256` for use in the header. it is probably worth doing this ASAP (i.e. here).

also, we tend not to have `get` in the names of functions and in any case "get now UTC" makes little sense when read as english. suggest `utcTime`.